features/step_definitions/active_docs_steps.rb

```diff
@@ -1,48 +1,60 @@
 # frozen_string_literal: true
 
-Given(/^the provider has (\d) active docs?$/) do |number_active_docs|
-  @active_docs = FactoryBot.create_list(:api_docs_service, number_active_docs.to_i, account: @provider, service: @provider.default_service)
+Given "the provider has {int} active doc(s)" do |number_active_docs|
+  @active_docs = FactoryBot.create_list(:api_docs_service, number_active_docs, account: @provider, service: @provider.default_service)
 end
 
-When /^I try to (create|update) the active docs( of the service)? with (in)?valid data$/ do |action, optional_scope, invalid|
+When "I try to {word} the active docs with {valid} data" do |action, valid|
+  action_active_docs(action, valid)
+end
+
+When "I try to {word} the active docs of the service with {valid} data" do |action, valid|
+  action_active_docs(action, valid, for_service: true)
+end
+
+def action_active_docs(action, valid, for_service: false)
```

Comment:
Should we define methods inside step definition files? Perhaps move to `World` or some helper module?

btw `step` is discouraged to be used for code reuse. But outside scope of this PR of course.
https://cucumber.io/docs/guides/anti-patterns/